.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+
+language: perl
+
+perl:
+  - "5.24"
+  - "5.22"
+  - "5.20"
+  - "5.18"
+  - "5.16"
+  - "5.14"
+  - "5.12"
+  - "5.10"
+  - "5.8"
+
+install:
+    - dzil authordeps --missing | cpanm --no-skip-satisfied || { cat ~/.cpanm/build.log ; false ; }
+    - dzil listdeps --author --missing | cpanm --no-skip-satisfied || { cat ~/.cpanm/build.log ; false ; }
```

Comment:
I've been wanting to integrate with Travis CI for some time now, but haven't gotten around to it.  Thank you for getting this set up!

Why is --no-skip-satisfied being specified explicitly?  I'll merge this anyway, but I'm curious if that can be removed since it's set by default, or if there's some gotcha you're avoiding.